app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,41 @@
  *= require font-awesome
  */
 
+#map-container { 
+  margin-top: 25px;
+  border-color: #fff;
+  border-style: solid;
+  border-radius: 16px 16px;
+  margin-top: 25px;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
+  width: 50px;
+  height: 400px;
+}
+
+#map { 
+  width: 100%;
+  height: 384px;
+
+}
+
+.search-location {
+ background-color: #fff;
+ background-image: none;
+ border: 1px solid #ccc;
+ border-radius: 4px;
+ -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+ box-shadow: inset 0 1px 1px rgba (0, 0, 0, 0.075);
+ color: #555;
+ font-size: 14px;
+ height: 34px;
+ line-height: 1.42857143;
+ padding: 6px 12px;
+ -o-transition: border-color ease-in-out 0.15s, box-shadow ease-in-out 0.15s;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Avoid vendor prefixes.